### PR TITLE
Simplified Chinese retranslation

### PR DESCRIPTION
**Pull request by Lzw104522773**
The original Chinese translation was not of very high quality, so I completely re-translated it.
The added install.tra is to avoid garbled installation prompts when selecting the Chinese option for installation.

### CODE REVIEW

File: RoT/setup-RoT.tp2 [hunk 1]
```diff
@@ -94,11 +94,13 @@ LANGUAGE ~Espanol~
...
 
-LANGUAGE ~Simplified Chinese (Optimization and translation By Youye)~
+//Replace the original Optimization and translation by Youye
+LANGUAGE ~Simplified Chinese (retranslation By Lzw104522773)~
 		 ~schinese~
+		 ~RoT/Language/schinese/install.tra~
 		 ~RoT/Language/schinese/scripts.tra~
 		 ~RoT/Language/schinese/journal.tra~
-		 ~RoT/Language/schinese/setup.tra~
+//		 ~RoT/Language/schinese/setup.tra~
```
Comment (Roberciiik) [inline RoT/setup-RoT.tp2:103]: You should include setup.tra file to load all items translations. To keep console prompts in proper encoding, put install.tra file after setup.tra in this list.

**Comment by Lzw104522773** [comment]
OK！

File: RoT/setup-RoT.tp2 (hunk 1 above)
Comment (Roberciiik) [reply RoT/setup-RoT.tp2]: @Lzw104522773 Please add install.tra as the last file in the list. Now setup.tra will overwrite the prompts from install.tra

**Comment by Lzw104522773** [comment]
OK

**Review by Roberciiik — APPROVED** [review]

**Pull request merged**